src/components/content/order/query/userOrderableServicesQuery.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query';
+import { ServiceCatalogService, ServiceVo } from '../../../../xpanse-api/generated';
+
+export default function userOrderableServicesQuery(category: ServiceVo.category, serviceName: string | undefined) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
we must check and handle this exception

Reply:
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/121531362/1a5c0c02-7132-4fa3-be87-03aeeaef977a)
Rename the  function name to 'UserOrderableServicesQuery' start with upper letter, the error is fixed.
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/121531362/3af126c3-c26d-401c-a4df-7ef0b0cca9d4)